doc/user/content/get-started/isolation-level.md

```diff
@@ -106,11 +106,58 @@ of the rows that were seen by T1.
 It’s important to note that the linearizable guarantee only applies to transactions (including single statement SQL
 queries which are implicitly single statement transactions), not to data written while ingesting from upstream sources.
 So if some piece of data has been fully ingested from an upstream source, then it is not guaranteed to appear in the
-next read transaction. See [real-time recency](https://github.com/MaterializeInc/materialize/issues/11531)
-and [strengthening correctness](https://github.com/MaterializeInc/materialize/issues/13107) for more details. If some
+next read transaction. See [real-time recency](#real-time-recency) and [strengthening correctness](https://github.com/MaterializeInc/materialize/issues/13107) for more details. If some
 piece of data has been fully ingested from an upstream source AND is included in the results of some read transaction
 THEN all subsequent read transactions are guaranteed to see that piece of data.
 
+### Real-time recency
+
+{{< private-preview />}}
+
+Materialize offers a form of "cross-system linearizability" known as real-time
+recency. When using real-time recency, all client-issued `SELECT` statements
+include at least all data visible to Materialize in any external source (i.e.,
+sources created with `CREATE SOURCE` that use `CONNECTION`s, such as Kafka,
+MySQL, and PostgreSQL sources) shortly after Materialize receives the query.
+This is what we mean by linearizable––the results are guaranteed to contain all
+visible data according to physical time.
+
+For example, real-time recency ensures that if you have just performed an
+`INSERT` into a PostgreSQL database that Materialize ingests as a source, all of
+your real-time recency queries will include the just-written data in their
+results.
+
+Note, however, that we only guarantee the results will contain the data visible
+to Materialize. We do not guarantee that we will consider _only_ the data
+visible at the time we receive the query. For instance, the rate at which
+Materialize ingests data might include additional data made visible after the
+timestamp we determined to be "real-time recent." Another example is that, due
```

Comment:
```suggestion
timestamp we determined to be "real-time recent". Another example is that, due
```

Reply:
Typically the period goes inside the quotation mark: https://www.hamilton.edu/academics/centers/writing/style/essentials/punctuation-of-quotations